### ISSUE TITLE
Multi-Label Text Classification

### ISSUE THREAD

**Issue by laxmimerit** [amaiya/ktrain#257]
Hi,
Thanks a lot for providing this beautiful wrapper. I am wondering, if anything in multi-label accuracy calculation got changed while upgrading from 0.18.5 to 0.21.3?

Because I had an accuracy of at 97% for my app while training with 0.18.5 but it got reduced to 70% in 0.21.3. I haven't changed anything in data or code. It's the same set of data and code.

It seems the accuracy calculation method got changed for multi-label text classification?

**Comment by amaiya** [comment]
This is a [bug in TensorFlow 2.2 and 2.3](https://github.com/tensorflow/tensorflow/issues/41114) and not a **ktrain** issue.  I've already reported it to them, and it is [fixed in TensorFlow 2.4](https://github.com/tensorflow/tensorflow/issues/41114#issuecomment-686778243).  

As a temporary workaround, you can compile model to use `binary_accuracy` instead of `accuracy`.  Note that this is just a metrics issue.  The loss and performance of the model is not affected (and accuracy is not the best metric for multilabel classification anyways.)
